pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -48,8 +48,9 @@ const (
 	// MemoryMin is memory.min for cgroup v2
 	MemoryMin string = "memory.min"
 	// MemoryHigh is memory.high for cgroup v2
-	MemoryHigh         string = "memory.high"
-	Cgroup2MaxCpuLimit string = "max"
+	MemoryHigh             string = "memory.high"
```

Comment:
should this also be renamed to add a prefix Cgroup2 for consistency?

Reply:
Done, also added the prefix to `MemoryMin` as well.